CONTRIBUTING.md

```diff
@@ -230,10 +182,8 @@ Once changie is installed and your PR is created, simply run `changie new` and c
 
 ## Submitting a Pull Request
 
-dbt Labs provides a CI environment to test changes to specific adapters, and periodic maintenance checks of `dbt-core` through Github Actions. For example, if you submit a pull request to the `dbt-redshift` repo, GitHub will trigger automated code checks and tests against Redshift.
-
 A `dbt-core` maintainer will review your PR. They may suggest code revision for style or clarity, or request that you add unit or integration test(s). These are good things! We believe that, with a little bit of help, anyone can contribute high-quality code.
-- First time contributors should note code checks + unit tests require a maintainer to approve.
 
+Automated tests run via GitHub Actions. If you're a first-time contributor, all tests (including code checks and unit tests) will require a maintainer to approve. Changes in the `dbt-core` repository trigger integration tests against Postgres, which a maintainer will enable by adding the `ok to test` label. dbt Labs also provides CI environments in which to test changes to other adapters, triggered by PRs in those adapters' repositories, as well as periodic maintenance checks of each adapter in concert with the latest `dbt-core` code changes.
```

Comment:
They run as part of the pre-commit hooks as well, but I'm not sure if that's worth calling out here or not.

Reply:
I don't think so since this section is specific to your pull request.